sequence_search/app.py

```diff
@@ -1,39 +1,26 @@
 import os
 from sanic import Sanic
 from sanic.response import json, file as fileResponse
-from .genesearch import Genesearch
+import urllib.parse
+from requests import get, post
 
 app = Sanic("sequence_search")
 
+
+@app.route('/api/v1/<api_path:path>', methods=["GET", "POST"])
+async def api_proxy(request, api_path):
+    api_url = urllib.parse.urljoin(os.environ['GENESEARCH_URL'], api_path)
```

Comment:
You probably want to sanitize / sanity-check the api_path before passing it on to the genesearch server.

Reply:
What happens if `GENESEARCH_URL` isn't in the environment? Some startup code might want to check on that and fail if it's not there. Or consider maintaining a config module that sets it up at server start time and keeps it in memory, otherwise if that environment variable changes during the run, it might have unexpected effects.

Reply:
If this is the main place things are happening, it's probably good to add some logging here about what paths are getting proxied, and whether the proxying works or not.